Assets/Vimeo/Scripts/Services/VimeoApi.cs

```diff
@@ -213,9 +244,17 @@ private void ResponseHandler(UnityWebRequest request)
                 } else {
                     SendError(request.url + " - " + request.downloadHandler.text, request.downloadHandler.text);
                 }
-            } else if (OnRequestComplete != null) {
-                OnRequestComplete(request.downloadHandler.text);
-            }
+            } else {
+                if (request.responseCode == 202) {
```

Comment:
what is a `202` status?

Reply:
The request has been accepted for processing, but the processing has not been completed.